router/router_system.go

```diff
@@ -63,6 +63,8 @@ func getSystemIps(c *gin.Context) {
 		interfaces = append(interfaces, targetIp)
 	}
 
+	log.Infof("Found %d IP addresses: %v", len(interfaces), interfaces)
```

Comment:
this should be a debug as it will flood the logs

Reply:
it's been removed